src/blosc2/schunk.py

```diff
@@ -725,6 +882,24 @@ def __setitem__(self, key, value):
         This method can also be used to append new data if :paramref:`key`.stop
         is greater than the SChunk nitems.
 
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> nchunks = 4
+        >>> chunk_size = 200 * 1000 * 4
+        >>> data = np.arange(nchunks * chunk_size // 4, dtype=np.int32)
+        >>> schunk = blosc2.SChunk(chunksize=chunk_size, data=data, cparams={"typesize": 4})
+        >>> # Create a new array of values to update the slice (values from 1000 to 1999 multiplied by 2)
+        >>> start_ = 1000
+        >>> stop = 2000
+        >>> new_values = np.arange(start_, stop, dtype=np.int32) * 2
+        >>> schunk.__setitem__(slice(start_, stop), new_values)
```

Comment:
Instead of `schunk.__setitem__` I would use `schunk[start_:stop] = new_values`. It's quicker : )